islandora_solr_config/islandora_solr_config.module

```diff
@@ -44,7 +44,7 @@ function islandora_solr_config_islandora_solr_secondary_display() {
       'class' => 'IslandoraSolrResultsCSV',
       'function' => 'printCSV',
       'description' => 'Download complete resultset as a CSV Spreadsheet',
-      'logo' => '<img src="' . $path . '/images/csv.png" class="secondary-display-csv">',
+      'logo' => '<img src="' . $path . '/images/csv.png" class="secondary-display-csv" alt="CSV Spreadsheet">',
```

Comment:
Done.
I also wondered about why this particular implementation didn't use theme(), but assumed there might be a reason so left it alone for now.